stdlib/grpc/src/main/java/org/ballerinalang/net/grpc/builder/BallerinaFileBuilder.java

```diff
@@ -304,6 +304,24 @@ private static Template compileTemplate(String defaultTemplateDir, String templa
 
             return result;
         });
+        handlebars.registerHelper("isNotNull", (object, options) -> {
+            CharSequence result;
+            if (object != null) {
```

Comment:
@daneshk If possible, we need to extract this to a separate method and use in both places.